src/posts/flypados3.md

```diff
@@ -0,0 +1,173 @@
+and overlays ruining your immersion.
+
+Our new integration now supports the aircraft position symbol for better visualization of where your aircraft is positioned on a relevant chart. Our unified experience means 
+that any chart you are viewing on the page will remain active even if you navigate away from the charts page. We've also included support for pinning relevant procedure maps 
+which are conveniently displayed on the flyPad's dashboard.
+
+Something we also hope you can look forward to is our planned Local Files support. This will allow users to access and use any image or PDF to be loaded onto the onboard EFB.
+
+![Navigraph Charts](/img/notam-images/flypados3/flypad-navigation.png)
+![Pinned 1](/img/notam-images/flypados3/pinned-charts.png)
+
+### Failure Simulation
+
+We have improved the UI for our failure simulations grouping them into ATA chapters. You can choose between two different views depending on how you want information to be 
+displayed. While we continuously work on improving the failures we plan to simulate this new UI allows us to easily expand failure simulations and provide a clean experience for 
+our users.
+
+![Failures Comfort](/img/notam-images/flypados3/flypad-failures.png)
+![Failures Compact](/img/notam-images/flypados3/flypad-failures-compact-view.png)
+
+### Online ATC
+
+ALT "tabbing" is not the most efficient use of your time when flying an IFR aircraft solo. The new online ATC page gives you a simplified overview of what frequencies are in 
+range and online in all stages of flight. You can easily swap between active or standby frequencies.
+
+We've also provided a text based output for ATIS, enabling our pilots to always have a quick reference to current ATIS information for departures and arrivals.
+
+![Online ATC](/img/notam-images/flypados3/flypad-online-atc-vatsim.png)
+
+### Presets
+
+### Customizable Lighting Presets
+
+Do you have that perfect cockpit lighting setup that you spend a few minutes on each flight configuring? Well now you can save it! We know that starting a flight in a cold and 
+dark state means you have a lot to do before you can get off the ground and climb into the sky. We hope this provides a nice little quality of life feature for our pilots.
+
+Our new presets' page allows you to save up to 8 custom presets, name them, and load them up instantaneously at any point during your flight.
+
+![Lighting Preset](/img/notam-images/flypados3/flypad-presets-lighting.png)
+
+#### Aircraft Presets
+
+We heard you liked having an AI co-pilot. This new feature gives your co-pilot control over "on-the-ground" actions up to your takeoff roll. 
+
+Our virtual co-pilot will sequentially performs the necessary setup tasks as in real life from ADIRs alignment to engine starts. Do note that even this feature is realistically 
```

Comment:
```suggestion
Our virtual co-pilot will sequentially perform the necessary setup tasks as in real life from ADIRs alignment to engine starts. Do note that even this feature is realistically 
```

Reply:
Fixed in a PR submitted into Val's fork.